### DISCUSSION TITLE
permission for bluetooth scanning?

### DISCUSSION THREAD

**Discussion by ray007** [cefsharp/CefSharp#4502] [Q&A]
I'm trying to use the experimental api for bluetooth scanning.

Trying in google chrome, the browser opens a dialog to ask the user for permission.

Trying in my CefSharp application, the call to `navigator.bluetooth.requestLEScan({...})` fails with
`Uncaught DOMException: User canceled the permission prompt.`.

Is permission granting through some API supported?
If so, could someone please point me in the right direction?

**Comment by amaitland** [comment]
https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=18792

**Reply by ray007** [reply]
Yes, thank you, I had already found that one too. But it didn't seem very helpful.
Is using the chrome runtime mode an option with CefSharp?
